README.md

```diff
@@ -4,36 +4,17 @@ CLANG:[![AWS-clang](https://codebuild.us-east-1.amazonaws.com/badges?uuid=eyJlbm
 
 # ioda-converters
 
-The intended way to use this repository is to install in `/usr/local` by default, and if you cannot write into `/usr/local` use the `--prefix` option of `ecbuild` to install in your home directory under `tools`.  Run the scripts from the place they are installed, not the source directory. That way the scripts can reference each other without providing a path.
+The converters can be built and tested using ioda-bundle. In ioda-bundle the build of the converters is disabled by default (for now) so you must enable the build using the BUILD_IODA_CONVERTERS directive. Here is an example:
 
-For example,
 ```
-ecbuild --prefix=$HOME/tools  /your/path/to/ioda-converters
-make
-make install
+git clone https://github.com/jcsda-internal/ioda-bundle
+cd ioda-bundle
+mkdir build
+cd build
+ecbuild -DBUILD_IODA_CONVERTERS=ON ..
```

Comment:
@srherbener I just realized we might want to add something here about 'srun', etc. so that on HPC platforms the tests with MPI don't run on the login nodes. Perhaps just run ctest on the ioda-converters tests for this example?

Reply:
@CoryMartin-NOAA 
You are right, parallel compile works for ioda-converters (no error), but failed only coding-norm ctest (that was the ctest I needed for my radar converter....)., other ctest passed though...

Orion-login-1[511] lliu$ make -j12
[ 38%] Built target gnssro_bufr2ioda
[100%] Built target bufr2nc_fortran.x
[100%] Built target test_iodaconv_gnssro_bufr_conv.x
[100%] Built target test_iodaconv_prepbufr_conv.x
[100%] Built target test_iodaconv_gnssro_conv.x
[100%] Built target test_iodaconv_mhs_conv.x
Orion-login-1[512] lliu$ ctest
Test project /work/noaa/marine/lliu/ioda/ioda_0322/ioda-converters/build
      Start  1: iodaconv_chem_coding_norms
 1/43 Test  #1: iodaconv_chem_coding_norms ...........***Failed    0.01 sec
      Start  2: iodaconv_gsi-ncdiag_coding_norms
 2/43 Test  #2: iodaconv_gsi-ncdiag_coding_norms .....***Failed    0.01 sec
      Start  3: iodaconv_land_coding_norms
 3/43 Test  #3: iodaconv_land_coding_norms ...........***Failed    0.01 sec
      Start  4: iodaconv_lib-python_coding_norms
 4/43 Test  #4: iodaconv_lib-python_coding_norms .....***Failed    0.01 sec

Reply:
i'm on Orion

Reply:
This now looks like you just built ioda-converters stand alone and not as part of the ioda-bundle like the instructions here? But the error before suggests you were using the ioda-bundle?

Reply:
that was correct...

Reply:
Ok so what are the modules you are using + what is the actual error at make time?

Reply:
there is no error at make time, but failed at ctest code-norms and only
I followed our JEDI academy to load modules on Orion:

export JEDI_OPT=/work/noaa/da/grubin/opt/modules
module use $JEDI_OPT/modulefiles/core
module load jedi/gnu-openmpi # GNU v8.3.0 with OpenMPI v4.0.2
module list

Reply:
then what error did you paste above?

Reply:
the failed ctest. I was trying to show that make passed, but ctest failed at code-norm

Reply:
So this is irrelevant? https://github.com/JCSDA-internal/ioda-converters/pull/481#issuecomment-805167555

Reply:
it is! because if you just follow this new readme to git clone the ioda-bundle, you will fail at both make -j4 (according to the new readme) and ctest.

I made a compromise to only git clone ioda-converters only, that will let pass the make but then will fail at ctest

Reply:
ok so it's not irrelevant, because you said that according to the new readme from this PR, it fails, so I was asking what the actual error was so we can troubleshoot.

Reply:
This is what I see on screen after make -j4, let me know how to see "real error", or what would you do to run this, let me know any of my steps was wrong please:
git clone https://github.com/jcsda-internal/ioda-bundle
cd ioda-bundle
mkdir build
cd build
ecbuild -DBUILD_IODA_CONVERTERS=ON ..
make -j4
ctest"

my error msg:
[ 81%] Building Fortran object oops/qg/model/CMakeFiles/qg.dir/qg_model_interface.F90.o
[ 81%] Linking CXX shared library ../../../lib/libqg.so
[ 81%] Built target qg
make: *** [all] Error 2

Reply:
My original point was when using make -j4, you need to paste more than just the last 4 lines to find where the actual error occurred in the compilation step. Check the complete output and look for other error messages or paste it from before 81%, like say 70% onward.

Reply:
or re-run with just make (no -j4)

Reply:
I'm sorry I missed the point, I'm rerunning it and saved to an output file so you could help take a look. thank you.